packages/components/src/stories/SquigglePlayground.stories.tsx

```diff
@@ -354,3 +354,36 @@ fn2 = {|e| e}
 y = x -> Tag.getAll`,
   },
 };
+
+export const Specification: Story = {
+  name: "Specification",
+  args: {
+    defaultCode: `validate(fn) = {
+      errors = List.upTo(2020, 2030)
+        -> List.map(
+          {|e| [Date(e), typeOf(fn(Date(e))) == "Distribution"]}
+        )
+        -> List.filter(
+          {|e| true}
+        )
+      "Has errors!"
+    }
+    
+    spec = Specification.make(
+      {
+        title: "Stock market over time",
+        description: "The S&P500 stock market price, over time.",
```

Comment:
I'm fine moving ``description`` to tags. I'm nervous about ``title``, because that should probably be mandatory. Seems weird to have a mandatory tag.